proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -157,15 +167,41 @@ message MavenConfig {
   repeated RuntimeConfig additional_runtimes = 3;
 }
 
+message SwiftConfig {
+  // RuntimeLibrary describes the runtime library dependency of the generated code.
+  message RuntimeLibrary {
```

Comment:
Would "dependency" make more sense than "library" here? (Not sure what was done for other package managers, so I'd keep it consistent with whatever was done for Maven)

Reply:
Dependency would work just as well, but all other registries use the RuntimeLibrary terminology